docs/pkp/authHelpers/addRemoveAuthMethods.md

```diff
@@ -0,0 +1,124 @@
+
+To write to the blockchain, the `LitContracts` instance must be created with a `ethers.Signer` that is authorized to sign transactions using the PKP. The [`@lit-protocol/pkp-ethers` package](https://js-sdk.litprotocol.com/modules/pkp_ethers_src.html) provides a convenient class, `PKPEthersWallet`, which can be used as a signer.
+
+## Initialize `PKPEthersWallet`
+
+`PKPEthersWallet` must be instantiated with an `AuthSig` or a `SessionSig` in order to authorize signing requests. To learn how to generate these signatures, refer to the [Authentication section](/SDK/Explanation/authentication).
+
+```js
+import { PKPEthersWallet } from '@lit-protocol/pkp-ethers';
+
+const pkpWallet = new PKPEthersWallet({
+  controllerAuthSig: '<Your AuthSig>',
+  // Or you can also pass in controllerSessionSigs
+  pkpPubKey: '<Your PKP public key>',
+  rpc: 'https://chain-rpc.litprotocol.com/http'
+});
+await pkpWallet.init();
+```
+
+To view more constructor options for `PKPEthersWallet`, check out the [API docs](https://js-sdk.litprotocol.com/interfaces/types_src.PKPEthersWalletProp.html).
+
+:::note
+
+**Passing `SessionSigs`**
+
+When generating session signatures for `PKPEthersWallet`, be sure to request the ability to execute Lit Actions by passing the following object in the `resourceAbilityRequests` array:
+
+```js
+{
+  resource: new LitActionResource('*'),
+  ability: LitAbility.LitActionExecution,
+}
+```
+
+:::
+
+## Initialize `LitContracts`
+
+Create an instance of `LitContracts` and pass in your `PKPEthersWallet`.
+
+```js
+import { LitContracts } from '@lit-protocol/contracts-sdk';
+
+const litContracts = new LitContracts({
+  signer: pkpWallet,
+});
+await litContracts.connect();
+```
+
+To view more constructor options for `LitContracts`, check out the [API docs](https://js-sdk.litprotocol.com/classes/contracts_sdk_src.LitContracts.html#constructor).
+
+## Construct the `AuthMethod` Object
+
+To add or remove an auth method, you must pass an `authMethod` object, which should have the following properties:
+
+- `authMethodType`: A number representing the type of auth method you want to add. Refer to the supported auth methods table [here](/pkp/authHelpers#existing-supported-auth-methods).
+- `id`: Bytes that represent a hash of a string that uniquely identifies the auth method
+- `userPubkey`: Public key of a WebAuthn credential (only required when using WebAuthn as an auth method)
+
+You can use the [`@lit-protocol/lit-auth-client` package](https://js-sdk.litprotocol.com/modules/lit_auth_client_src.html) to first authenticate the user and then derive the unique auth method `id`.
+
+To authenticate the user, check out these [guides](/SDK/Explanation/authentication/sessionSigs/authMethods/overview) that walk you through the process of obtaining authentication material from a `LitAuthClient` auth provider. Once the user is authenticated, you can call the `getAuthMethodId` utility function.
+
```

Comment:
Should we also explicitly add that you may register your own custom auth method for use in lit actions? which would not be supported by the auth client?

Reply:
p.s. i removed the getAuthMethodId stuff